server/src/frontend_api.rs

```diff
@@ -1,3 +1,250 @@
-use actix_web::web;
+use actix_web::{
+    get,
+    web::{self, Json},
+};
+use unleash_types::{
+    client_features::{ClientFeatures, Payload},
+    frontend::{EvaluatedToggle, EvaluatedVariant, FrontendResult},
+};
+use unleash_yggdrasil::{Context, EngineState};
 
-pub fn configure_frontend_api(_cfg: &mut web::ServiceConfig) {}
+use crate::types::{EdgeJsonResult, EdgeProvider, EdgeToken};
+
+#[get("/proxy/all")]
+async fn get_frontend_features(
+    edge_token: EdgeToken,
+    features_source: web::Data<dyn EdgeProvider>,
+    context: web::Query<Context>,
+) -> EdgeJsonResult<FrontendResult> {
+    let client_features = features_source.get_client_features(edge_token);
+    let context = context.into_inner();
+
+    let evaluated_features = resolve_frontend_features(client_features, context);
+
+    Ok(Json(FrontendResult {
+        toggles: evaluated_features,
+    }))
+}
+
+#[actix_web::post("/proxy/all")]
```

Comment:
I'm assuming we can `use` post above instead of simplify this by using `post` only, which would also be more consistent with the `get` in the route above.

Reply:
Yeah, nice catch, updated